release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -146,6 +146,10 @@ For more information, see xref:../updating/updating-cluster-prepare.adoc#updatin
 ==== Minimum required permissions for Microsoft Azure to install and delete an {product-title} cluster
 In {product-title} {product-version}, instead of using the built-in roles, you can now define your custom roles to include the minimum required permissions for Microsoft Azure to install and delete an {product-title} cluster. These permissions are available for installer-provisioned infrastructure and user-provisioned infrastructure.
 
+[id="ocp-4-13-updating-clusters-migrating-cluster-to-multi-arch"]
+==== Convenient single-architecture to multi-architecture payload migration
+{product-title} {product-version} introduces the `oc adm upgrade --to-multi-arch` command, which allows you to migrate your current cluster with single-architecture compute machines to a cluster with multi-architecture compute machines. By updating to a multi-architecture, manifest-listed payload, you can add mixed architecture compute nodes to your cluster.
```

Comment:
I think the main thing with 4.13 is that multi-arch [becomes GA][1] (but only for some control-plane/compute arch tuples?  I don't see that called out in the release notes at the moment).  I'd expect `--to-multi-arch` to get called out in a section discussing the GA-ness, with "There's also a new, more convenient way to transition existing single-arch clusters..." or something like that.

[1]: https://github.com/openshift/openshift-docs/pull/58100/commits/bc7a76d30737aada469af97e0438f9b87e1c648b#diff-5a1bef860ba4f103b12cdc2db661140fa691167a8b50abea180008754482951cR1180

Reply:
@wking what do you think about adding this info above https://github.com/openshift/openshift-docs/pull/58844/files or below it? that's the only multi-arch-related content so far. Thoughts?

Reply:
@shellyyang1989 @wking I've received approval from the multi-arch writer on the content, and there doesn't appear to be more multi-arch context from their side. Is there something else you'd like me to add, or are we set for now?

Reply:
Got the all-clear from @wking to pass for peer review. @shellyyang1989 plz let me know if you have any concerns. Will wait for the LGTM before merging :)

Reply:
The content looks good to me.